src/Generating/StructGen.cpp

```diff
@@ -183,6 +183,14 @@ void cStructGenTrees::ApplyTreeImage(
 				CASE_TREE_OVERWRITTEN_BLOCKS:
 				{
 					a_ChunkDesc.SetBlockTypeMeta(itr->m_RelX, itr->m_RelY, itr->m_RelZ, itr->m_BlockType, itr->m_BlockMeta);
+					// If grass is below our tree, turn it to dirt
```

Comment:
Tree! Tree? My dear LogicParrot, this is a *StructGen*.

Still, I dislike the whole commit-fetch-rebase-push, especially for little things like comments, so I shan't mind if you don't bother.